android/brave_java_sources.gni

```diff
@@ -451,7 +451,6 @@ brave_java_sources = [
   "../../brave/android/java/org/chromium/chrome/browser/vpn/BraveVpnNativeWorker.java",
   "../../brave/android/java/org/chromium/chrome/browser/vpn/BraveVpnObserver.java",
   "../../brave/android/java/org/chromium/chrome/browser/vpn/DisconnectVpnBroadcastReceiver.java",
-  "../../brave/android/java/org/chromium/chrome/browser/vpn/activities/BraveVpnParentActivity.java",
```

Comment:
did we remove `BraveVpnParentActivity.java`?

Reply:
@tapanmodh it's included as a part of `android/java/org/chromium/chrome/browser/vpn/billing/sources.gni` as one of the import depends on billing client